[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36898:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36898)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36898)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36898] will exit after 30000ms
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36898] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36898] close 1 servers!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36898] worker disconnect!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] worker:36898 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] new worker:36920 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36901:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36901)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36901)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36901] will exit after 30000ms
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36901] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36901] close 1 servers!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36901] worker disconnect!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] worker:36901 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] new worker:36921 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36897:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36897)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36897)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36897] will exit after 30000ms
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36897] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36897] close 1 servers!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36897] worker disconnect!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] worker:36897 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] new worker:36922 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36902:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36902)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36902)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36902] will exit after 30000ms
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36902] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36902] close 1 servers!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36902] worker disconnect!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] worker:36902 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] new worker:36923 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36900:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36900)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36900)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36900] will exit after 30000ms
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36900] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36900] close 1 servers!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36900] worker disconnect!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] worker:36900 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] new worker:36924 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36899:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36899)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36899)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36899] will exit after 30000ms
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36899] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36899] close 1 servers!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36899] worker disconnect!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] worker:36899 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] new worker:36925 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36903:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36903)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36903)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36903] will exit after 30000ms
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36903] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36903] close 1 servers!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36903] worker disconnect!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] worker:36903 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] new worker:36926 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36896:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36896)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36896)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36896] will exit after 30000ms
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36896] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36896] close 1 servers!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [graceful:worker:36896] worker disconnect!
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] worker:36896 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:09 GMT+0800 (CST)] [cfork:master:36870] new worker:36927 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36922:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36922)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36922)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36922] will exit after 30000ms
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36922] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36922] close 1 servers!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36922] worker disconnect!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] worker:36922 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] new worker:36944 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36920:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36920)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36920)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36920] will exit after 30000ms
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36920] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36920] close 1 servers!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36920] worker disconnect!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] worker:36920 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] new worker:36946 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36921:uncaughtException] throw error 1 times
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36926:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36921)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36921)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36921] will exit after 30000ms
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36926)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36926)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36921] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36921] close 1 servers!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36926] will exit after 30000ms
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36921] worker disconnect!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36926] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36926] close 1 servers!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36926] worker disconnect!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] worker:36921 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] new worker:36947 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] worker:36926 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] new worker:36948 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36923:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36923)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36923)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36923] will exit after 30000ms
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36923] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36923] close 1 servers!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [graceful:worker:36923] worker disconnect!
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] worker:36923 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:12 GMT+0800 (CST)] [cfork:master:36870] new worker:36949 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36925:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36925)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36925)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36925] will exit after 30000ms
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36925] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36925] close 1 servers!
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36925] worker disconnect!
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [cfork:master:36870] worker:36925 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [cfork:master:36870] new worker:36950 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36927:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36927)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36927)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36927] will exit after 30000ms
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36927] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36927] close 1 servers!
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36927] worker disconnect!
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [cfork:master:36870] worker:36927 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [cfork:master:36870] new worker:36953 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36924:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36924)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36924)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36924] will exit after 30000ms
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36924] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36924] close 1 servers!
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [graceful:worker:36924] worker disconnect!
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [cfork:master:36870] worker:36924 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:13 GMT+0800 (CST)] [cfork:master:36870] new worker:36955 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36946:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36946)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36946)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36946] will exit after 30000ms
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36946] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36946] close 1 servers!
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36946] worker disconnect!
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [cfork:master:36870] worker:36946 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [cfork:master:36870] new worker:36979 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36944:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36944)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36944)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36944] will exit after 30000ms
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36944] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36944] close 1 servers!
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [graceful:worker:36944] worker disconnect!
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [cfork:master:36870] worker:36944 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:15 GMT+0800 (CST)] [cfork:master:36870] new worker:36982 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36949:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36949)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36949)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36949] will exit after 30000ms
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36949] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36949] close 1 servers!
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36949] worker disconnect!
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [cfork:master:36870] worker:36949 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [cfork:master:36870] new worker:36983 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36947:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36947)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36947)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36947] will exit after 30000ms
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36947] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36947] close 1 servers!
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36947] worker disconnect!
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [cfork:master:36870] worker:36947 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [cfork:master:36870] new worker:36984 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36950:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36950)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36950)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36950] will exit after 30000ms
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36950] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36950] close 1 servers!
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [graceful:worker:36950] worker disconnect!
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [cfork:master:36870] worker:36950 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:16 GMT+0800 (CST)] [cfork:master:36870] new worker:36985 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36953:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36953)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36953)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36953] will exit after 30000ms
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36953] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36953] close 1 servers!
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36953] worker disconnect!
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [cfork:master:36870] worker:36953 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [cfork:master:36870] new worker:36988 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36955:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36955)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36955)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36955] will exit after 30000ms
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36955] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36955] close 1 servers!
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [graceful:worker:36955] worker disconnect!
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [cfork:master:36870] worker:36955 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:17 GMT+0800 (CST)] [cfork:master:36870] new worker:36989 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:18 GMT+0800 (CST)] [graceful:worker:36948:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36948)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36948)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:18 GMT+0800 (CST)] [graceful:worker:36948] will exit after 30000ms
[Sun Mar 19 2017 12:31:18 GMT+0800 (CST)] [graceful:worker:36948] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:18 GMT+0800 (CST)] [graceful:worker:36948] close 1 servers!
[Sun Mar 19 2017 12:31:18 GMT+0800 (CST)] [graceful:worker:36948] worker disconnect!
[Sun Mar 19 2017 12:31:18 GMT+0800 (CST)] [cfork:master:36870] worker:36948 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:18 GMT+0800 (CST)] [cfork:master:36870] new worker:36998 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36984:uncaughtException] throw error 1 times
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36979:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36979)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36979)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36979] will exit after 30000ms
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36979] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36979] close 1 servers!
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36979] worker disconnect!
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [cfork:master:36870] worker:36979 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [cfork:master:36870] new worker:37001 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36984)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36984)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36984] will exit after 30000ms
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36984] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36984] close 1 servers!
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36984] worker disconnect!
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [cfork:master:36870] worker:36984 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [cfork:master:36870] new worker:37002 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36982:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36982)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36982)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36982] will exit after 30000ms
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36982] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36982] close 1 servers!
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [graceful:worker:36982] worker disconnect!
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [cfork:master:36870] worker:36982 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:19 GMT+0800 (CST)] [cfork:master:36870] new worker:37003 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36983:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36983)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36983)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36983] will exit after 30000ms
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36983] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36983] close 1 servers!
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36983] worker disconnect!
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [cfork:master:36870] worker:36983 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [cfork:master:36870] new worker:37004 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36985:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36985)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36985)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36985] will exit after 30000ms
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36985] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36985] close 1 servers!
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36985] worker disconnect!
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [cfork:master:36870] worker:36985 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [cfork:master:36870] new worker:37005 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36988:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36988)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36988)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36988] will exit after 30000ms
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36988] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36988] close 1 servers!
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36988] worker disconnect!
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [cfork:master:36870] worker:36988 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [cfork:master:36870] new worker:37006 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36989:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36989)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36989)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36989] will exit after 30000ms
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36989] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36989] close 1 servers!
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [graceful:worker:36989] worker disconnect!
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [cfork:master:36870] worker:36989 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:20 GMT+0800 (CST)] [cfork:master:36870] new worker:37007 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:21 GMT+0800 (CST)] [graceful:worker:36998:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36998)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:36998)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:21 GMT+0800 (CST)] [graceful:worker:36998] will exit after 30000ms
[Sun Mar 19 2017 12:31:21 GMT+0800 (CST)] [graceful:worker:36998] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:21 GMT+0800 (CST)] [graceful:worker:36998] close 1 servers!
[Sun Mar 19 2017 12:31:21 GMT+0800 (CST)] [graceful:worker:36998] worker disconnect!
[Sun Mar 19 2017 12:31:21 GMT+0800 (CST)] [cfork:master:36870] worker:36998 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:21 GMT+0800 (CST)] [cfork:master:36870] new worker:37014 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37001:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37001)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37001)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37001] will exit after 30000ms
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37001] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37001] close 1 servers!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37001] worker disconnect!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] worker:37001 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] new worker:37017 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37003:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37003)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37003)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37003] will exit after 30000ms
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37003] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37003] close 1 servers!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37003] worker disconnect!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] worker:37003 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] new worker:37018 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37002:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37002)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37002)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37002] will exit after 30000ms
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37002] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37002] close 1 servers!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37002] worker disconnect!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] worker:37002 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] new worker:37019 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37004:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37004)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37004)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37004] will exit after 30000ms
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37004] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37004] close 1 servers!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37004] worker disconnect!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] worker:37004 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] new worker:37020 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37005:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37005)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37005)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37005] will exit after 30000ms
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37005] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37005] close 1 servers!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37005] worker disconnect!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] worker:37005 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] new worker:37021 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37006:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37006)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37006)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37006] will exit after 30000ms
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37006] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37006] close 1 servers!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37006] worker disconnect!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] worker:37006 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] new worker:37022 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37007:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37007)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37007)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37007] will exit after 30000ms
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37007] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37007] close 1 servers!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [graceful:worker:37007] worker disconnect!
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] worker:37007 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:22 GMT+0800 (CST)] [cfork:master:36870] new worker:37023 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:23 GMT+0800 (CST)] [graceful:worker:37014:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37014)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37014)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:23 GMT+0800 (CST)] [graceful:worker:37014] will exit after 30000ms
[Sun Mar 19 2017 12:31:23 GMT+0800 (CST)] [graceful:worker:37014] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:23 GMT+0800 (CST)] [graceful:worker:37014] close 1 servers!
[Sun Mar 19 2017 12:31:23 GMT+0800 (CST)] [graceful:worker:37014] worker disconnect!
[Sun Mar 19 2017 12:31:23 GMT+0800 (CST)] [cfork:master:36870] worker:37014 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:23 GMT+0800 (CST)] [cfork:master:36870] new worker:37030 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37017:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37017)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37017)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37017] will exit after 30000ms
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37017] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37017] close 1 servers!
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37017] worker disconnect!
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [cfork:master:36870] worker:37017 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [cfork:master:36870] new worker:37034 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37018:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37018)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37018)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37018] will exit after 30000ms
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37018] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37018] close 1 servers!
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37018] worker disconnect!
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [cfork:master:36870] worker:37018 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [cfork:master:36870] new worker:37035 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37019:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37019)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37019)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37019] will exit after 30000ms
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37019] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37019] close 1 servers!
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37019] worker disconnect!
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [cfork:master:36870] worker:37019 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [cfork:master:36870] new worker:37036 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37020:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37020)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37020)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37020] will exit after 30000ms
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37020] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37020] close 1 servers!
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37020] worker disconnect!
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [cfork:master:36870] worker:37020 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [cfork:master:36870] new worker:37037 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37021:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37021)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37021)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:24 GMT+0800 (CST)] [graceful:worker:37021] will exit after 30000ms
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37021] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37021] close 1 servers!
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37021] worker disconnect!
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [cfork:master:36870] worker:37021 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [cfork:master:36870] new worker:37038 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37022:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37022)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37022)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37022] will exit after 30000ms
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37022] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37022] close 1 servers!
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37022] worker disconnect!
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [cfork:master:36870] worker:37022 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [cfork:master:36870] new worker:37039 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37023:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37023)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37023)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37023] will exit after 30000ms
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37023] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37023] close 1 servers!
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [graceful:worker:37023] worker disconnect!
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [cfork:master:36870] worker:37023 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:25 GMT+0800 (CST)] [cfork:master:36870] new worker:37040 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37030:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37030)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37030)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37030] will exit after 30000ms
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37030] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37030] close 1 servers!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37030] worker disconnect!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] worker:37030 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] new worker:37052 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37035:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37035)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37035)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37035] will exit after 30000ms
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37035] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37035] close 1 servers!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37035] worker disconnect!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] worker:37035 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] new worker:37054 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37037:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37037)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37037)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37037] will exit after 30000ms
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37037] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37037] close 1 servers!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37037] worker disconnect!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] worker:37037 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] new worker:37055 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37034:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37034)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37034)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37034] will exit after 30000ms
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37034] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37034] close 1 servers!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37034] worker disconnect!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] worker:37034 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] new worker:37056 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37036:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37036)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37036)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37036] will exit after 30000ms
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37036] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37036] close 1 servers!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [graceful:worker:37036] worker disconnect!
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] worker:37036 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:26 GMT+0800 (CST)] [cfork:master:36870] new worker:37057 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37038:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37038)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37038)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37038] will exit after 30000ms
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37038] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37038] close 1 servers!
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37038] worker disconnect!
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [cfork:master:36870] worker:37038 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [cfork:master:36870] new worker:37058 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37039:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37039)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37039)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37039] will exit after 30000ms
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37039] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37039] close 1 servers!
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37039] worker disconnect!
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [cfork:master:36870] worker:37039 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [cfork:master:36870] new worker:37059 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37040:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37040)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37040)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37040] will exit after 30000ms
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37040] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37040] close 1 servers!
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37040] worker disconnect!
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [cfork:master:36870] worker:37040 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [cfork:master:36870] new worker:37060 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37052:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37052)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37052)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37052] will exit after 30000ms
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37052] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37052] close 1 servers!
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [graceful:worker:37052] worker disconnect!
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [cfork:master:36870] worker:37052 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:27 GMT+0800 (CST)] [cfork:master:36870] new worker:37061 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37055:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37055)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37055)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37055] will exit after 30000ms
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37055] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37055] close 1 servers!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37055] worker disconnect!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker:37055 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] new worker:37062 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37054:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37054)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37054)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37054] will exit after 30000ms
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37054] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37054] close 1 servers!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37054] worker disconnect!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker:37054 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] new worker:37063 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37056:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37056)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37056)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37056] will exit after 30000ms
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37056] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37056] close 1 servers!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37056] worker disconnect!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker:37056 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] new worker:37064 fork (state: none)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37057:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37057)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37057)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37057] will exit after 30000ms
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37057] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37057] close 1 servers!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37057] worker disconnect!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker:37057 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker died too fast (total 60 disconnect, 0 unexpected exit)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] don't fork new work (refork: true)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37058:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37058)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37058)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37058] will exit after 30000ms
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37058] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37058] close 1 servers!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37058] worker disconnect!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker:37058 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker died too fast (total 61 disconnect, 0 unexpected exit)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] don't fork new work (refork: true)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37060:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37060)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37060)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37060] will exit after 30000ms
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37060] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37060] close 1 servers!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37060] worker disconnect!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker:37060 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker died too fast (total 62 disconnect, 0 unexpected exit)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] don't fork new work (refork: true)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37059:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37059)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37059)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37059] will exit after 30000ms
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37059] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37059] close 1 servers!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37059] worker disconnect!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker:37059 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker died too fast (total 63 disconnect, 0 unexpected exit)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] don't fork new work (refork: true)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37061:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37061)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37061)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37061] will exit after 30000ms
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37061] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37061] close 1 servers!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [graceful:worker:37061] worker disconnect!
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker:37061 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] worker died too fast (total 64 disconnect, 0 unexpected exit)
[Sun Mar 19 2017 12:31:28 GMT+0800 (CST)] [cfork:master:36870] don't fork new work (refork: true)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37063:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37063)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37063)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37063] will exit after 30000ms
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37063] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37063] close 1 servers!
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37063] worker disconnect!
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] worker:37063 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] worker died too fast (total 65 disconnect, 0 unexpected exit)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] don't fork new work (refork: true)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37062:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37062)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37062)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37062] will exit after 30000ms
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37062] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37062] close 1 servers!
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37062] worker disconnect!
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] worker:37062 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] worker died too fast (total 66 disconnect, 0 unexpected exit)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] don't fork new work (refork: true)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37064:uncaughtException] throw error 1 times
{ EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37064)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 7001,
  name: 'EADDRINUSEError' }
EADDRINUSEError: bind EADDRINUSE null:7001 (uncaughtException throw 1 times on pid:37064)
    at Object.exports._errnoException (util.js:1028:11)
    at exports._exceptionWithHostPort (util.js:1051:20)
    at cb (net.js:1326:16)
    at rr (internal/cluster/child.js:111:12)
    at Worker.send (internal/cluster/child.js:78:7)
    at process.onInternalMessage (internal/cluster/utils.js:42:8)
    at emitTwo (events.js:111:20)
    at process.emit (events.js:192:7)
    at process.nextTick (internal/child_process.js:757:12)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37064] will exit after 30000ms
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37064] close server#0, _connections: 0
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37064] close 1 servers!
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [graceful:worker:37064] worker disconnect!
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] worker:37064 disconnect (suicide: true, state: disconnected, isDead: false)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] worker died too fast (total 67 disconnect, 0 unexpected exit)
[Sun Mar 19 2017 12:31:29 GMT+0800 (CST)] [cfork:master:36870] don't fork new work (refork: true)
(node:36870) DeprecationWarning: worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36898] kill timeout, exit now.
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36901] kill timeout, exit now.
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [cfork:master:36870] worker:36898 exit (code: 1, suicide: true, state: dead, isDead: true, isExpected: true)
2017-03-19 12:31:39,399 ERROR 36870 nodejs.AppWorkerDiedError: [master] App Worker#3:36898 died (code: 1, signal: 1, suicide: true, state: dead), current workers: ["1","2","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67"]
    at Master.onAppExit (/Users/chenmingjia/Documents/Boolan-Egg/Boolan-Egg/node_modules/egg-cluster/lib/master.js:280:19)
    at emitOne (events.js:96:13)
    at Master.emit (events.js:189:7)
    at Messenger.sendToMaster (/Users/chenmingjia/Documents/Boolan-Egg/Boolan-Egg/node_modules/egg-cluster/lib/utils/messenger.js:122:17)
    at Messenger.send (/Users/chenmingjia/Documents/Boolan-Egg/Boolan-Egg/node_modules/egg-cluster/lib/utils/messenger.js:87:12)
    at EventEmitter.cluster.on (/Users/chenmingjia/Documents/Boolan-Egg/Boolan-Egg/node_modules/egg-cluster/lib/master.js:190:22)
    at emitThree (events.js:121:20)
    at EventEmitter.emit (events.js:195:7)
    at ChildProcess.worker.process.once (internal/cluster/master.js:189:13)
    at Object.onceWrapper (events.js:291:19)
name: 'AppWorkerDiedError'
pid: 36870
hostname: chenminjiadeMBP

[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36897] kill timeout, exit now.
2017-03-19 12:31:39,402 ERROR 36870 [master] worker start fail, exit now
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37062:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37064:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37062)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37062)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37064)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37064)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37061:uncaughtException] throw error 2 times
{ ECONNRESETError:[Sun Mar 19 2017 12:31:40 GMT+0800 (CST)] [graceful:worker:36912] kill timeout, exit now.
[Sun Mar 19 2017 12:31:40 GMT+0800 (CST)] [graceful:worker:36908] kill timeout, exit now.
[Sun Mar 19 2017 12:31:40 GMT+0800 (CST)] [graceful:worker:36911] kill timeout, exit now.
[Sun Mar 19 2017 12:31:40 GMT+0800 (CST)] [graceful:worker:36910] kill timeout, exit now.
[Sun Mar 19 2017 12:31:41 GMT+0800 (CST)] [graceful:worker:36913] kill timeout, exit now.
[Sun Mar 19 2017 12:31:41 GMT+0800 (CST)] [graceful:worker:36915] kill timeout, exit now.
[Sun Mar 19 2017 12:31:41 GMT+0800 (CST)] [graceful:worker:36914] kill timeout, exit now.
[Sun Mar 19 2017 12:31:41 GMT+0800 (CST)] [graceful:worker:36917] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36938] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36935] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36937] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36933] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36943] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36940] kill timeout, exit now.
[Sun Mar 19 2017 12:31:44 GMT+0800 (CST)] [graceful:worker:36939] kill timeout, exit now.
[Sun Mar 19 2017 12:31:45 GMT+0800 (CST)] [graceful:worker:36936] kill timeout, exit now.
[Sun Mar 19 2017 12:31:47 GMT+0800 (CST)] [graceful:worker:36957] kill timeout, exit now.
[Sun Mar 19 2017 12:31:47 GMT+0800 (CST)] [graceful:worker:36961] kill timeout, exit now.
[Sun Mar 19 2017 12:31:47 GMT+0800 (CST)] [graceful:worker:36960] kill timeout, exit now.
[Sun Mar 19 2017 12:31:47 GMT+0800 (CST)] [graceful:worker:36958] kill timeout, exit now.
[Sun Mar 19 2017 12:31:48 GMT+0800 (CST)] [graceful:worker:36964] kill timeout, exit now.
[Sun Mar 19 2017 12:31:48 GMT+0800 (CST)] [graceful:worker:36962] kill timeout, exit now.
[Sun Mar 19 2017 12:31:48 GMT+0800 (CST)] [graceful:worker:36963] kill timeout, exit now.
[Sun Mar 19 2017 12:31:49 GMT+0800 (CST)] [graceful:worker:36972] kill timeout, exit now.
[Sun Mar 19 2017 12:31:50 GMT+0800 (CST)] [graceful:worker:36993] kill timeout, exit now.
[Sun Mar 19 2017 12:31:50 GMT+0800 (CST)] [graceful:worker:36992] kill timeout, exit now.
[Sun Mar 19 2017 12:31:50 GMT+0800 (CST)] [graceful:worker:36994] kill timeout, exit now.
[Sun Mar 19 2017 12:31:51 GMT+0800 (CST)] [graceful:worker:36995] kill timeout, exit now.
[Sun Mar 19 2017 12:31:51 GMT+0800 (CST)] [graceful:worker:36996] kill timeout, exit now.
[Sun Mar 19 2017 12:31:51 GMT+0800 (CST)] [graceful:worker:36997] kill timeout, exit now.
[Sun Mar 19 2017 12:31:51 GMT+0800 (CST)] [graceful:worker:36999] kill timeout, exit now.
[Sun Mar 19 2017 12:31:51 GMT+0800 (CST)] [graceful:worker:37000] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37008] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37010] kill timeout, exit now.
[Sun Mar 19 2017 12:31:53 GMT+0800 (CST)] [graceful:worker:37009] kill timeout, exit now.
[Sun Mar 19 2017 12:31:53 GMT+0800 (CST)] [graceful:worker:37012] kill timeout, exit now.
[Sun Mar 19 2017 12:31:53 GMT+0800 (CST)] [graceful:worker:37011] kill timeout, exit now.
[Sun Mar 19 2017 12:31:53 GMT+0800 (CST)] [graceful:worker:37013] kill timeout, exit now.
[Sun Mar 19 2017 12:31:53 GMT+0800 (CST)] [graceful:worker:37015] kill timeout, exit now.
[Sun Mar 19 2017 12:31:53 GMT+0800 (CST)] [graceful:worker:37016] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37024] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37025] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37027] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37028] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37026] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37031] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37029] kill timeout, exit now.
[Sun Mar 19 2017 12:31:56 GMT+0800 (CST)] [graceful:worker:37032] kill timeout, exit now.
[Sun Mar 19 2017 12:31:57 GMT+0800 (CST)] [graceful:worker:37041] kill timeout, exit now.
[Sun Mar 19 2017 12:31:57 GMT+0800 (CST)] [graceful:worker:37044] kill timeout, exit now.
[Sun Mar 19 2017 12:31:57 GMT+0800 (CST)] [graceful:worker:37046] kill timeout, exit now.
ow 2 times on pid:36947)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36946:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36949:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36946)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36946)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36926:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36949)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36949)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36926)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36926)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37023:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36982:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36920:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36982)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36982)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36920)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36920)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36944:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36944)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36944)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36924:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36900:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36979:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36924)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36924)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36988:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36979)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36979)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36988)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36988)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36927:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36927)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36927)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36983)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36983)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36902:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36950:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36921:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36950)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36950)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37023)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37023)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36921)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36921)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36900)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36900)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36922:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36902)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36902)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36922)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36925:uncaughtException] throw error 2 times
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36922)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36925)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36925)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37039:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37039)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37039)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37054:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37036:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37054)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37054)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37036)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37036)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37052:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37052)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37052)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37037:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37037)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37037)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37022:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37035:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37035)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37035)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37022)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37022)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37021:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37020:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37021)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37020)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37020)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37019:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37019)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37019)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37063:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37059:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37063)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37063)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37059)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37059)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37034:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37040:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37040)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37040)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37034)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37034)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37021)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37038:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37017:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37038)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37038)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37017)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37017)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37018:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37006:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37030:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37018)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37018)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37006)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37030)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37030)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37006)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37056:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37004:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37004)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37004)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37056)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37056)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37007:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37007)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37007)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37055:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37055)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37055)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37002:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37003:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37002)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37002)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37003)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37003)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37014:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37014)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37014)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37058:uncaughtException] throw error 2 times
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:37005:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37058)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37058)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37005)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:37005)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36989:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36989)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36989)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36923:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36923)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36923)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36899:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36899)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36899)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36896:uncaughtException] throw error 2 times
{ ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36896)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  name: 'ECONNRESETError' }
ECONNRESETError: read ECONNRESET (uncaughtException throw 2 times on pid:36896)
    at exports._errnoException (util.js:1028:11)
    at TCP.onread (net.js:572:26)
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36902] kill timeout, exit now.
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36900] kill timeout, exit now.
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36899] kill timeout, exit now.
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36903] kill timeout, exit now.
[Sun Mar 19 2017 12:31:39 GMT+0800 (CST)] [graceful:worker:36896] kill timeout, exit now.
[Sun Mar 19 2017 12:31:42 GMT+0800 (CST)] [graceful:worker:36922] kill timeout, exit now.
[Sun Mar 19 2017 12:31:42 GMT+0800 (CST)] [graceful:worker:36920] kill timeout, exit now.
[Sun Mar 19 2017 12:31:42 GMT+0800 (CST)] [graceful:worker:36921] kill timeout, exit now.
[Sun Mar 19 2017 12:31:42 GMT+0800 (CST)] [graceful:worker:36926] kill timeout, exit now.
[Sun Mar 19 2017 12:31:42 GMT+0800 (CST)] [graceful:worker:36923] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36925] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36927] kill timeout, exit now.
[Sun Mar 19 2017 12:31:43 GMT+0800 (CST)] [graceful:worker:36924] kill timeout, exit now.
[Sun Mar 19 2017 12:31:45 GMT+0800 (CST)] [graceful:worker:36946] kill timeout, exit now.
[Sun Mar 19 2017 12:31:45 GMT+0800 (CST)] [graceful:worker:36944] kill timeout, exit now.
[Sun Mar 19 2017 12:31:46 GMT+0800 (CST)] [graceful:worker:36949] kill timeout, exit now.
[Sun Mar 19 2017 12:31:46 GMT+0800 (CST)] [graceful:worker:36947] kill timeout, exit now.
[Sun Mar 19 2017 12:31:46 GMT+0800 (CST)] [graceful:worker:36950] kill timeout, exit now.
[Sun Mar 19 2017 12:31:47 GMT+0800 (CST)] [graceful:worker:36953] kill timeout, exit now.
[Sun Mar 19 2017 12:31:47 GMT+0800 (CST)] [graceful:worker:36955] kill timeout, exit now.
[Sun Mar 19 2017 12:31:48 GMT+0800 (CST)] [graceful:worker:36948] kill timeout, exit now.
[Sun Mar 19 2017 12:31:49 GMT+0800 (CST)] [graceful:worker:36979] kill timeout, exit now.
[Sun Mar 19 2017 12:31:49 GMT+0800 (CST)] [graceful:worker:36984] kill timeout, exit now.
[Sun Mar 19 2017 12:31:49 GMT+0800 (CST)] [graceful:worker:36982] kill timeout, exit now.
[Sun Mar 19 2017 12:31:50 GMT+0800 (CST)] [graceful:worker:36983] kill timeout, exit now.
[Sun Mar 19 2017 12:31:50 GMT+0800 (CST)] [graceful:worker:36985] kill timeout, exit now.
[Sun Mar 19 2017 12:31:50 GMT+0800 (CST)] [graceful:worker:36988] kill timeout, exit now.
[Sun Mar 19 2017 12:31:50 GMT+0800 (CST)] [graceful:worker:36989] kill timeout, exit now.
[Sun Mar 19 2017 12:31:51 GMT+0800 (CST)] [graceful:worker:36998] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37001] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37003] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37002] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37004] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37005] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37006] kill timeout, exit now.
[Sun Mar 19 2017 12:31:52 GMT+0800 (CST)] [graceful:worker:37007] kill timeout, exit now.
[Sun Mar 19 2017 12:31:53 GMT+0800 (CST)] [graceful:worker:37014] kill timeout, exit now.
[Sun Mar 19 2017 12:31:54 GMT+0800 (CST)] [graceful:worker:37017] kill timeout, exit now.
[Sun Mar 19 2017 12:31:54 GMT+0800 (CST)] [graceful:worker:37018] kill timeout, exit now.
[Sun Mar 19 2017 12:31:54 GMT+0800 (CST)] [graceful:worker:37019] kill timeout, exit now.
[Sun Mar 19 2017 12:31:54 GMT+0800 (CST)] [graceful:worker:37020] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37021] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37022] kill timeout, exit now.
[Sun Mar 19 2017 12:31:55 GMT+0800 (CST)] [graceful:worker:37023] kill timeout, exit now.
[Sun Mar 19 2017 12:31:56 GMT+0800 (CST)] [graceful:worker:37030] kill timeout, exit now.
[Sun Mar 19 2017 12:31:56 GMT+0800 (CST)] [graceful:worker:37035] kill timeout, exit now.
[Sun Mar 19 2017 12:31:56 GMT+0800 (CST)] [graceful:worker:37037] kill timeout, exit now.
[Sun Mar 19 2017 12:31:56 GMT+0800 (CST)] [graceful:worker:37034] kill timeout, exit now.
[Sun Mar 19 2017 12:31:56 GMT+0800 (CST)] [graceful:worker:37036] kill timeout, exit now.
[Sun Mar 19 2017 12:31:57 GMT+0800 (CST)] [graceful:worker:37038] kill timeout, exit now.
[Sun Mar 19 2017 12:31:57 GMT+0800 (CST)] [graceful:worker:37039] kill timeout, exit now.
[Sun Mar 19 2017 12:31:57 GMT+0800 (CST)] [graceful:worker:37040] kill timeout, exit now.
[Sun Mar 19 2017 12:31:57 GMT+0800 (CST)] [graceful:worker:37052] kill timeout, exit now.
[Sun Mar 19 2017 12:31:58 GMT+0800 (CST)] [graceful:worker:37055] kill timeout, exit now.
[Sun Mar 19 2017 12:31:58 GMT+0800 (CST)] [graceful:worker:37054] kill timeout, exit now.
[Sun Mar 19 2017 12:31:58 GMT+0800 (CST)] [graceful:worker:37056] kill timeout, exit now.
[Sun Mar 19 2017 12:31:58 GMT+0800 (CST)] [graceful:worker:37057] kill timeout, exit now.
[Sun Mar 19 2017 12:31:58 GMT+0800 (CST)] [graceful:worker:37058] kill timeout, exit now.
[Sun Mar 19 2017 12:31:58 GMT+0800 (CST)] [graceful:worker:37060] kill timeout, exit now.
[Sun Mar 19 2017 12:31:58 GMT+0800 (CST)] [graceful:worker:37059] kill timeout, exit now.
[Sun Mar 19 2017 12:31:58 GMT+0800 (CST)] [graceful:worker:37061] kill timeout, exit now.
[Sun Mar 19 2017 12:31:59 GMT+0800 (CST)] [graceful:worker:37063] kill timeout, exit now.
[Sun Mar 19 2017 12:31:59 GMT+0800 (CST)] [graceful:worker:37062] kill timeout, exit now.
[Sun Mar 19 2017 12:31:59 GMT+0800 (CST)] [graceful:worker:37064] kill timeout, exit now.
